domain/model/modelmigration/import.go

```diff
@@ -159,7 +159,7 @@ func (i *importOperation) Execute(ctx context.Context, model description.Model)
 	if model.CloudCredential() != nil {
 		cred.Name = model.CloudCredential().Name()
 		cred.Cloud = model.CloudCredential().Cloud()
-		cred.Owner = model.CloudCredential().Owner()
+		cred.Owner, _ = coreuser.NewName(model.CloudCredential().Owner())
```

Comment:
We shouldn't ignore this error.

Reply:
Looks like that slipped through with the IDE automatic refactoring, have grepped for other instances and it looks like that was the only one.